### ISSUE TITLE
Don't hide source language of projects

### ISSUE THREAD

**Issue by gwaldvogel** [translate/pootle#4778]
When using Pootle the source language of a project will not be shown for translation.

Example:
Project source language is en_GB, there is a Qt .ts file called en_GB which only contains untranslated strings. Pootle will not show British English as a translatable language for that project.

This is a problem when the source language translation is used to correct mistakes (spelling, grammar, etc.) made by the developer.

It would be nice to be able to show the source language as a translatable language.

**Comment by unho** [comment]
It is possible to use Pootle to handle this scenario. You just have to use the `templates` language to be used as source language and also enable the source language by using its actual language code. For example use `templates` and `en_GB`.

**Comment by gwaldvogel** [comment]
So I basically duplicate en_GB.ts and call the copy templates.ts and it will work?

**Comment by unho** [comment]
Yes, the idea behind is to copy `en_GB.ts` to `templates.ts`. But might not work because most likely you already have created the project and it matched `en_GB` to be `templates`. But try adding the missing `templates.ts` and creating the `en_GB` translation project to see if it works.

**Comment by unho** [comment]
@gwaldvogel Can you please share with us if the proposed way actually worked for you?
